brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/Project.java

```diff
@@ -64,4 +76,63 @@ public String getIdentifier() {
     public String getName() {
         return name;
     }
+
+    private static FundingSource findFundingsSource(FundingSources fundingSources, Project project) {
+        return fundingSources.getSources()
+                   .stream()
+                   .filter(source -> source.getName().containsValue(project.name))
+                   .findFirst()
+                   .orElse(null);
+    }
+
+    private static int getSeparatorIndexPrioritizingColon(int colonIndex, int slashIndex, int commaIndex) {
```

Comment:
This method name does not appear to match the actual logic. Is this working as intended?

Reply:
Agree, will change name to getSeparatorIndex()